friends/src/components/Friend.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+//issues with array not repopulating on page refresh. returns an undefined error - TypeError: Cannot read property 'name' of undefined
+
+const Friend = props => {
+  const id = props.match.params.id
+  const friend = props.friends.find(friend => {
+    console.log(friend, id);
+    return `${friend.id}` === id;
+  })
+  return (
+    <div>
+        <h3>{friend.name}</h3>
```

Comment:
this is a good opportunity for object deconstruction.
```
const {name, age, email, id} = friend;
```
```suggestion
        <h3>{name}</h3>
```